### ISSUE TITLE
The reproduction of other methods

### ISSUE THREAD

**Issue by nuistji** [johncai117/DAMSL#3]
Excellent work! Does the code contain all the methods you have presented in the article? Can you tell me how to reimplement the TransFT method and its augmentation version?

**Comment by johncai117** [comment]
Thanks! Yes, it contains all the methods aside from those indicated with results taken from other papers.

For "Transductive fine-tuning", it is already implemented by removing .eval() in finetune.py during testing. This ensures that the model recalculates the batch-norm of the entire support + query set during testing, rather than freezing the batch norm with training parameters. 

"In addition to the above setup, we also experimented with the transductive setting, where the model classifies the entire test set at once. In our transductive experiments, information was shared between the test samples via batch normalization [9]. In our non-transductive experiments, batch normalization statistics were computed using all of the training samples and a single test sample. We note that Finn et al. [4] use transduction for evaluating MAML."

You can refer to Nichol et al (2018) "On First-Order Meta-Learning Algorithms".

**Comment by nuistji** [comment]
Thank you! This helps me a lot!
